src/platform/tls_openssl.c

```diff
@@ -2516,15 +2516,16 @@ CxPlatTlsInitialize(
     }
 
     //
-    // The fuzzers can't handle tls messages that span datagrams it appears, and ML-KEM pushes us
-    // over the limit, so if we have an ALPN of "fuzz", mimic what schannel offers in terms
-    // of supported key share groups
-    // NOTE: The AlpnBufferLength needs to be at least 5 here, as the first byte indicates the
-    // length of the following alpn identifier
-    if (TlsContext->AlpnBufferLength >= 5 &&
-        !strncmp((const char *)&TlsContext->AlpnBuffer[1], "fuzz", 4)) {
-        SSL_set1_groups_list(TlsContext->Ssl, "secp256r1:x25519");
-    }
+    // Both the fuzzer and the HandshakeSpecificLossPattern tests have some
+    // issues with larger key shares as introduced by ML-KEM support in openssl
+    // The former doesn't expect Client/Server hellos to span multiple udp datagrams
+    // and hits a buffer space assertion failure, while the latter times out on loss
+    // recovery.  So for now mimic the key shares that schannel offers to work around
+    // that
+    //
+    // TODO: Remove this when the above tests are tolerant of addition of ML-KEM keyshares
+    //
+    SSL_set1_groups_list(TlsContext->Ssl, "secp256r1:x25519");
```

Comment:
I'd suggest trying `"?*X25519:?secp256r1 / ?X448:?secp384r1:?secp521r1 / ?ffdhe2048:?ffdhe3072"` as this will set the same configuration as in default settings but without enabling the hybrid key share .